src/app.py

```diff
@@ -422,7 +481,42 @@ def param_search_index(self, plural_entity_type):
                                                 , es_url=oss_base_url
                                                 , query=json_query_dict
                                                 , request_params={'filter_path':'hits.hits._source'})
-
+            
+            accepted_s3_domains = ["hm-api-responses.s3.amazonaws.com"]
+
+            manifest_list = []
```

Comment:
Due to the fact that multiple search endpoints use the same handling for manifest file generation, we should create a shared method rather than repeating the code.